airbyte-integrations/connectors/source-stripe/source_stripe/streams.py

```diff
@@ -700,19 +700,19 @@ def read_records(self, sync_mode: SyncMode, stream_slice: Optional[Mapping[str,
             items_next_pages = super().read_records(sync_mode=SyncMode.full_refresh, stream_slice=stream_slice, **kwargs)
         yield from chain(items, items_next_pages)
 
-    def stream_slices(
-        self, sync_mode: SyncMode, cursor_field: Optional[List[str]] = None, stream_state: Optional[Mapping[str, Any]] = None
-    ) -> Iterable[Optional[Mapping[str, Any]]]:
-        parent_stream_slices = self.parent.stream_slices(
-            sync_mode=SyncMode.full_refresh, cursor_field=cursor_field, stream_state=stream_state
-        )
-        for stream_slice in parent_stream_slices:
-            parent_records = self.parent.read_records(
-                sync_mode=SyncMode.full_refresh, cursor_field=cursor_field, stream_slice=stream_slice, stream_state=stream_state
-            )
-            for record in parent_records:
-                self.logger.info(f"Fetching parent stream slices for stream {self.name}.")
-                yield {"parent": record}
+    # def stream_slices(
```

Comment:
the stripe connector was modified to add prints. This had a significant impact on the conenctor's performance because it prints one line per parent record. this should be reverted before updating the connector again